test/stories/ledger.test.ts.snap

```diff
@@ -175,3 +175,18 @@ exports[`test/stories/ledger Level 3: Host and collective with different currenc
 | PLATFORM_TIP            | CREDIT | 10000000  | JPY      | 90000                | USD          | 0          | OC Inc | Ben    | OC Inc |            | false    |
 | PLATFORM_TIP            | DEBIT  | -10000000 | JPY      | -90000               | USD          | 0          | Ben    | OC Inc | NULL   |            | false    |"
 `;
+
+exports[`test/stories/ledger Level 4: Refund added funds️ Refund added funds with different collectives 1`] = `
+"
+| kind        | type   | amount | paymentFee | To      | From    | Host | Settlement | isRefund |
+| ----------- | ------ | ------ | ---------- | ------- | ------- | ---- | ---------- | -------- |
+| ADDED_FUNDS | CREDIT | 10000  | 0          | ESLint  | Webpack | OSC  |            | false    |
+| ADDED_FUNDS | DEBIT  | -10000 | 0          | Webpack | ESLint  | OSC  |            | false    |"
+`;
+
+exports[`test/stories/ledger Level 4: Refund added funds️ Refund added funds with same collective 1`] = `
+"
+| kind        | type   | amount | paymentFee | To     | From   | Host | Settlement | isRefund |
+| ----------- | ------ | ------ | ---------- | ------ | ------ | ---- | ---------- | -------- |
+| ADDED_FUNDS | CREDIT | 10000  | 0          | ESLint | ESLint | OSC  |            | false    |"
+`;
```

Comment:
Here we're only snapshotting the data before the refund (aka the setup data). Since the goal is to test the refund, it'd be way more useful to snapshot **after** the refund, to make sure we include both the transaction and its refund.

Reply:
@Betree : I've added the snapshot after the refund. Let me know if you see any further issues.

Reply:
That looks good to me, I'll let @znarf share a final feedback with this new snapshot in place